test/__helpers/e2e-system-test.ts

```diff
@@ -198,6 +202,8 @@ export async function e2eTestApp(ctx: ReturnType<typeof setupE2EContext>) {
     `
   )
 
+  await ctx.spawn(['npx', 'prisma2', `geneate`])
```

Comment:
@Weakky I'm surprised I needed this... 🤔 I guess it doesn't happen on post install anymore?